backend/controller/cronjobs/cronjobs.go

```diff
@@ -168,8 +168,8 @@ func (s *Service) CreatedOrReplacedDeloyment(ctx context.Context, newDeploymentK
 	_ = s.syncJobsWithNewDeploymentKey(ctx, optional.Some(newDeploymentKey))
 }
 
-// syncJobs is run periodically via a scheduled task
-func (s *Service) syncJobs(ctx context.Context) (time.Duration, error) {
+// SyncJobs is run periodically via a scheduled task
+func (s *Service) SyncJobs(ctx context.Context) (time.Duration, error) {
```

Comment:
I think further to what we discussed earlier re. data model abstractions, we should also pull out all of the "logic" of the controller into a separate package such that the controller itself contains very little logic.

Not for this PR, just a note.